lib/helpers.bash

```diff
@@ -18,49 +18,49 @@ case "$OSTYPE" in
   'darwin'*) BASH_IT_SED_I_PARAMETERS=(-i "")
 esac
 
-function _command_exists ()
-{
-  _about 'checks for existence of a command'
-  _param '1: command to check'
-  _param '2: (optional) log message to include when command not found'
-  _example '$ _command_exists ls && echo exists'
-  _group 'lib'
-  local msg="${2:-Command '$1' does not exist!}"
-  if type -t "$1" &> /dev/null
-  then
-	  return 0
-  else
-	  _log_warning "$msg"
-	  return 1
-  fi
+function _command_exists() {
+	_about 'checks for existence of a command'
+	_param '1: command to check'
+	_param '2: (optional) log message to include when command not found'
+	_example '$ _command_exists ls && echo exists'
+	_group 'lib'
+	local msg="${2:-Command '$1' does not exist\!}"
+	if type "$1" &> /dev/null; then
+		return "$?"
+	else
+		_log_debug "$msg"
```

Comment:
why change to a debug log?

Reply:
There's not a thing wrong. There's no problem to "warn" about. I don't even know why there's logging in there to begin with. 

I would have removed the logging entirely, but months ago someone(?) mentioned switching it to debug log instead of removing it so I just did that.